Remora.Discord.Commands/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -246,7 +246,7 @@ public static IServiceCollection AddDiscordCommands
 
         serviceCollection.TryAddSingleton<ExecutionEventCollectorService>();
 
-        serviceCollection.TryAddScoped<FeedbackService>();
+        serviceCollection.TryAddScoped<IFeedbackService, FeedbackService>();
```

Comment:
```suggestion
        serviceCollection.TryAddScoped<FeedbackService>();
        serviceCollection.TryAddScoped<IFeedbackService>(s => s.GetRequiredService<FeedbackService>());
```
To keep backwards compatibility.